src/herbpy/herbrobot.py

```diff
@@ -225,7 +225,7 @@ def __init__(self, left_arm_sim, right_arm_sim, right_ft_sim,
                      TSRPlanner(delegate_planner=actual_planner),
                      self.cbirrt_planner),
             # Special purpose meta-planner.
-            NamedPlanner(delegate_planner=actual_planner),
+            NamedPlanner(delegate_planner=Sequence(actual_planner, self.cbirrt_planner)),
```

Comment:
This allows calls to ```PlanToNamedConfiguration``` to use a randomized planner (in this case ```CBiRRT```) when snap planner and optimizers have failed. We are seeing that when planning out of tight quarters near objects using CHOMP the resulting trajectory fails to get pushed out of collision.  I think this change is important, especially if a user has only CHOMP installed and not trajopt, but we will pull it out and put it into a separate PR.